ormconfig.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  type: 'mysql',
+  host: process.env.DB_HOST,
+  port: 3306,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  synchronize: true,
```

Comment:
Check more about this on TypeORM docs since this setting cannot be used in production so this should be coming from environment variables 👍